tools/data-api/main.go

```diff
@@ -9,14 +9,18 @@ import (
 	"github.com/mitchellh/cli"
 )
 
+const definitionsDirectory = "../../api-definitions/"
+
 func main() {
 	logger := hclog.New(hclog.DefaultOptions)
 	logger.SetLevel(hclog.Trace)
 
 	c := cli.NewCLI("data-api", "1.0.0")
 	c.Args = os.Args[1:]
 	c.Commands = map[string]cli.CommandFactory{
-		"serve": func() (cli.Command, error) {
+		"serve": commands.NewServeCommand(definitionsDirectory),
+		// TODO hook this up
+		"serve-watch": func() (cli.Command, error) {
 			return commands.ServeCommand{
 				Log: logger,
 			}, nil
```

Comment:
Probably better to be explicit this isn't supported yet?

```suggestion
			return nil, fmt.Error("TODO: implement me")
```